packages/frama-clang/frama-clang.0.0.13/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Frama-Clang, Frama-C's C++ front-end"
+description:"""
+Frama-Clang is a plug-in of Frama-C that allows parsing C++ files.
+It is based on clang/LLVM
+"""
+maintainer: "virgile.prevosto@cea.fr"
+authors: [
+  "David Cok"
+  "Virgile Prevosto"
+  "Franck Védrine"
+]
+homepage: "https://frama-c.com/frama-clang.html"
+license: "LGPL-2.1-only"
+dev-repo: "git+https://git.frama-c.com/pub/frama-clang.git"
+bug-reports: "https://git.frama-c.com/pub/frama-clang/-/issues"
+tags: [
+  "Frama-C"
+  "C++"
+]
+
+url {
+  src: "https://frama-c.com/download/frama-clang-0.0.13.tar.gz"
+  checksum: [ "sha512=d4fbfc170b43e7486fd6920363fe5ed73cfd6e1d537e6f64790f5cf9384e6d0c73eb626ee8c79d5492ffc559eec8ad649ed6ed2e5caedb519ab74c781f34f6ff" ]
+}
+
+build: [
+  ["sed" "-i" "-e" "s|llvm-config|%{conf-llvm:config}%|g" "configure"]
```

Comment:
`sed -i` works differently one GNU, Busybox (Alpine) and macOS. Only `-i.<suffix>` is supported on all platforms
```suggestion
  ["sed" "-i.bak" "-e" "s|llvm-config|%{conf-llvm:config}%|g" "configure"]
```

Reply:
You're right, thanks for reminding me about sed's portability issues.